core-bundle/src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -442,13 +442,13 @@ public function showAll()
 			$label = $GLOBALS['TL_LANG']['MOD']['files'][0];
 		}
 
-		$icon = $GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['icon'] ?: 'filemounts.svg';
+		$icon = $GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['icon'] ?? 'filemounts.svg';
```

Comment:
Same here, if icon was set to `false` or `''` this behaves different now.

Reply:
Fixed in 1550fd769ce350112719147253a37736f5c7cd79.